demo/demo-schema/src/main/java/io/servicecomb/demo/CodeFirstRestTemplate.java

```diff
@@ -57,6 +59,11 @@ public void testCodeFirst(RestTemplate template, String microserviceName, String
       //            testCodeFirstRawJsonString(template, cseUrlPrefix);
       testCodeFirstSayHello(template, cseUrlPrefix);
       testCodeFirstReduce(template, cseUrlPrefix);
+
+      // TODO: highway unsupported until JAV-394 completed
+      if(transport.equals("rest")) {
```

Comment:
if transport is "", that means auto select a transport, that maybe highway

Reply:
Checked，DemoConst.transports = new String[] {"highway", "rest", Const.ANY_TRANSPORT}; 
Const.ANY_TRANSPORT = "";
So，testModelFieldIgnore will be call once , I think that's enough ?

Reply:
ok